metaflow/plugins/kfp/kfp.py

```diff
@@ -593,6 +593,11 @@ def _set_container_labels(
             container_op.add_pod_annotation(f"{prefix}/experiment", self.experiment)
         if self.tags and len(self.tags) > 0:
             for tag in self.tags:
+                tag_name = f"{prefix}/tag_{tag}"
+                if len(tag_name) > 63:
+                    raise ValueError(
+                        f"Tag name {tag_name} must be no more than 63 characters"
+                    )
```

Comment:
ideally a unit test for this ... or run a Flow and look for it

Reply:
Yes definitely. Adding...

Reply:
@cloudw gentle reminder of this.